sdk/test/UnitTests/Custom/RegionEndpointTests.cs

```diff
@@ -6,7 +6,6 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
 using Amazon.Auth.AccessControlPolicy;
```

Comment:
nit: seems we can also remove this statement `using Amazon.Auth.AccessControlPolicy;`